frontend/src/components/Modals/ShoutoutDeleteModal 2.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
```

Comment:
Why is this file "ShoutoutDeleteModal 2" (and its corresponding CSS here)? It appears to be the same as the other "ShoutoutDeleteModal" file.

Reply:
Thank you for pointing this out, I'm not sure what happened!